`PG_SLEEP` returns `cannot write xcontent for unknown value of type class io.crate.expression.symbol.Literal`

### CrateDB version

6.4.0

### CrateDB setup information

Standard single node

### Problem description

Running `PG_SLEEP` returns a `SQLParseException`.

### Steps to Reproduce

```sql
SELECT PG_SLEEP(1);
```

### Actual Result

Successful execution when run via the PostgreSQL protocol, but the below exception via HTTP (Admin UI/`crash`):

```
SQLParseException[cannot write xcontent for unknown value of type class io.crate.expression.symbol.Literal]

io.crate.exceptions.SQLParseException: cannot write xcontent for unknown value of type class io.crate.expression.symbol.Literal
	at io.crate.exceptions.SQLExceptions.esToCrateException(SQLExceptions.java:201)
	at io.crate.exceptions.SQLExceptions.prepareForClientTransmission(SQLExceptions.java:190)
	at io.crate.rest.action.SqlHttpHandler.sendResponse(SqlHttpHandler.java:168)
	at io.crate.rest.action.SqlHttpHandler.lambda$channelRead0$0(SqlHttpHandler.java:129)
	at java.base/java.util.concurrent.CompletableFuture.uniWhenComplete(CompletableFuture.java:884)
	at java.base/java.util.concurrent.CompletableFuture.uniWhenCompleteStage(CompletableFuture.java:908)
	at java.base/java.util.concurrent.CompletableFuture.whenComplete(CompletableFuture.java:2384)
	at io.crate.rest.action.SqlHttpHandler.channelRead0(SqlHttpHandler.java:126)
	at io.crate.rest.action.SqlHttpHandler.channelRead0(SqlHttpHandler.java:80)
	at io.netty.channel.SimpleChannelInboundHandler.channelRead(SimpleChannelInboundHandler.java:99)
	at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:357)
	at io.netty.handler.codec.MessageToMessageDecoder.channelRead(MessageToMessageDecoder.java:107)
	at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:357)
	at io.crate.protocols.http.HttpBlobHandler.channelRead0(HttpBlobHandler.java:191)
	at io.netty.channel.SimpleChannelInboundHandler.channelRead(SimpleChannelInboundHandler.java:99)
	at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:357)
	at io.crate.auth.HttpAuthUpstreamHandler.handleHttpChunk(HttpAuthUpstreamHandler.java:165)
	at io.crate.auth.HttpAuthUpstreamHandler.channelRead0(HttpAuthUpstreamHandler.java:96)
	at io.netty.channel.SimpleChannelInboundHandler.channelRead(SimpleChannelInboundHandler.java:99)
	at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:357)
	at io.netty.handler.codec.http.HttpContentDecoder.decode(HttpContentDecoder.java:170)
	at io.netty.handler.codec.http.HttpContentDecoder.decode(HttpContentDecoder.java:48)
	at io.netty.handler.codec.MessageToMessageDecoder.channelRead(MessageToMessageDecoder.java:91)
	at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:357)
	at io.netty.handler.codec.ByteToMessageDecoder.fireChannelRead(ByteToMessageDecoder.java:361)
	at io.netty.handler.codec.ByteToMessageDecoder.channelRead(ByteToMessageDecoder.java:325)
	at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:357)
	at io.netty.handler.timeout.IdleStateHandler.channelRead(IdleStateHandler.java:288)
	at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:355)
	at io.netty.channel.ChannelInboundHandlerAdapter.channelRead(ChannelInboundHandlerAdapter.java:93)
	at org.elasticsearch.transport.netty4.Netty4InboundStatsHandler.channelRead(Netty4InboundStatsHandler.java:99)
	at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:357)
	at io.netty.channel.DefaultChannelPipeline$HeadContext.channelRead(DefaultChannelPipeline.java:1429)
	at io.netty.channel.DefaultChannelPipeline.fireChannelRead(DefaultChannelPipeline.java:918)
	at io.netty.channel.epoll.AbstractEpollStreamChannel$EpollStreamUnsafe.epollInReady(AbstractEpollStreamChannel.java:804)
	at io.netty.channel.epoll.AbstractEpollChannel$AbstractEpollUnsafe.handle(AbstractEpollChannel.java:487)
	at io.netty.channel.epoll.EpollIoHandler$DefaultEpollIoRegistration.handle(EpollIoHandler.java:349)
	at io.netty.channel.epoll.EpollIoHandler.processReady(EpollIoHandler.java:546)
	at io.netty.channel.epoll.EpollIoHandler.run(EpollIoHandler.java:491)
	at io.netty.channel.SingleThreadIoEventLoop.runIo(SingleThreadIoEventLoop.java:225)
	at io.netty.channel.SingleThreadIoEventLoop.run(SingleThreadIoEventLoop.java:196)
	at io.netty.util.concurrent.SingleThreadEventExecutor$5.run(SingleThreadEventExecutor.java:1195)
	at io.netty.util.internal.ThreadExecutorMap$2.run(ThreadExecutorMap.java:74)
	at java.base/java.lang.Thread.run(Thread.java:1474)
Caused by: java.lang.IllegalArgumentException: cannot write xcontent for unknown value of type class io.crate.expression.symbol.Literal
	at org.elasticsearch.common.xcontent.XContentBuilder.unknownValue(XContentBuilder.java:805)
	at org.elasticsearch.common.xcontent.XContentBuilder.value(XContentBuilder.java:773)
	at io.crate.rest.action.ResultToXContentBuilder.addRow(ResultToXContentBuilder.java:124)
	at io.crate.rest.action.RestResultSetReceiver.setNextRow(RestResultSetReceiver.java:68)
	at io.crate.session.RetryOnFailureResultReceiver.setNextRow(RetryOnFailureResultReceiver.java:65)
	at io.crate.session.RowConsumerToResultReceiver.consumeIt(RowConsumerToResultReceiver.java:86)
	at io.crate.session.RowConsumerToResultReceiver.accept(RowConsumerToResultReceiver.java:61)
	at io.crate.execution.engine.InterceptingRowConsumer.accept(InterceptingRowConsumer.java:63)
	at io.crate.execution.engine.collect.CollectTask.lambda$new$1(CollectTask.java:113)
	at org.elasticsearch.common.util.concurrent.PriorityRunnable$SimplePriorityRunnable.run(PriorityRunnable.java:87)
	at org.elasticsearch.common.util.concurrent.EsExecutors$DirectExecutor.execute(EsExecutors.java:160)
	at io.crate.execution.engine.collect.MapSideDataCollectOperation.launch(MapSideDataCollectOperation.java:63)
	at io.crate.execution.engine.collect.CollectTask.lambda$new$0(CollectTask.java:115)
	at java.base/java.util.concurrent.CompletableFuture.uniWhenComplete(CompletableFuture.java:884)
	at java.base/java.util.concurrent.CompletableFuture$UniWhenComplete.tryFire(CompletableFuture.java:862)
	at java.base/java.util.concurrent.CompletableFuture.postComplete(CompletableFuture.java:531)
	at java.base/java.util.concurrent.CompletableFuture.complete(CompletableFuture.java:2206)
	at io.crate.execution.engine.collect.CollectTask.lambda$start$0(CollectTask.java:212)
	at java.base/java.util.concurrent.CompletableFuture.uniWhenComplete(CompletableFuture.java:884)
	at java.base/java.util.concurrent.CompletableFuture.uniWhenCompleteStage(CompletableFuture.java:908)
	at java.base/java.util.concurrent.CompletableFuture.whenComplete(CompletableFuture.java:2384)
	at io.crate.execution.engine.collect.CollectTask.start(CollectTask.java:210)
	at io.crate.execution.jobs.RootTask.start(RootTask.java:220)
	at io.crate.execution.jobs.RootTask.start(RootTask.java:205)
	at io.crate.execution.engine.JobLauncher.setupTasks(JobLauncher.java:264)
	at io.crate.execution.engine.JobLauncher.execute(JobLauncher.java:162)
	at io.crate.planner.operators.LogicalPlanner.executeNodeOpTree(LogicalPlanner.java:738)
	at io.crate.planner.operators.LogicalPlanner.doExecute(LogicalPlanner.java:676)
	at io.crate.planner.operators.LogicalPlanner.execute(LogicalPlanner.java:641)
	at io.crate.planner.operators.LogicalPlan.executeOrFail(LogicalPlan.java:216)
	at io.crate.planner.Plan.execute(Plan.java:75)
	at io.crate.session.Session.singleExec(Session.java:930)
	at io.crate.session.Session.execute(Session.java:602)
	at io.crate.rest.action.SqlHttpHandler.executeSimpleRequest(SqlHttpHandler.java:254)
	at io.crate.rest.action.SqlHttpHandler.handleSQLRequest(SqlHttpHandler.java:216)
	at io.crate.rest.action.SqlHttpHandler.channelRead0(SqlHttpHandler.java:125)
	... 36 more
```

### Expected Result

Successful execution

--- a/docs/appendices/release-notes/6.3.6.rst
+++ b/docs/appendices/release-notes/6.3.6.rst
@@ -46,4 +46,5 @@ series.
 Fixes
 =====
 
-None
+- Fixed an issue that would throw an error when using :ref:`pg_sleep()
+  <scalar-pg_sleep>` function in the ``SELECT`` clause.

--- a/docs/appendices/release-notes/6.4.1.rst
+++ b/docs/appendices/release-notes/6.4.1.rst
@@ -46,4 +46,5 @@ series.
 Fixes
 =====
 
-None
+- Fixed an issue that would throw an error when using :ref:`pg_sleep()
+  <scalar-pg_sleep>` function in the ``SELECT`` clause.

--- a/server/src/main/java/io/crate/expression/scalar/postgres/PgSleepFunction.java
+++ b/server/src/main/java/io/crate/expression/scalar/postgres/PgSleepFunction.java
@@ -22,7 +22,6 @@
 package io.crate.expression.scalar.postgres;
 
 import io.crate.data.Input;
-import io.crate.expression.symbol.Literal;
 import io.crate.metadata.FunctionName;
 import io.crate.metadata.FunctionType;
 import io.crate.metadata.Functions;
@@ -64,9 +63,9 @@ public class PgSleepFunction extends Scalar<Object, Double> {
             try {
                 Thread.sleep((long) (duration * 1000.0));
             } catch (InterruptedException e) {
-                return Literal.NULL;
+                return null;
             }
         }
-        return Literal.NULL;
+        return null;
     }
 }

--- a/server/src/test/java/io/crate/expression/scalar/PGSleepFunctionTest.java
+++ b/server/src/test/java/io/crate/expression/scalar/PGSleepFunctionTest.java
@@ -26,21 +26,19 @@ import static org.assertj.core.api.Assertions.assertThatThrownBy;
 
 import org.junit.Test;
 
-import io.crate.expression.symbol.Literal;
-
 public class PGSleepFunctionTest extends ScalarTestCase {
 
     @Test
     public void test_pg_sleep() {
         long before = System.currentTimeMillis();
-        assertEvaluate("pg_catalog.pg_sleep(0.1)", Literal.NULL);
+        assertEvaluateNull("pg_catalog.pg_sleep(0.1)");
         long after = System.currentTimeMillis();
         assertThat(after - before >= 100).isTrue();
     }
 
     @Test
     public void test_pg_sleep_wrong_arguments() {
-        assertThatThrownBy(() -> assertEvaluate("pg_sleep(1, 1)", Literal.NULL))
+        assertThatThrownBy(() -> assertEvaluateNull("pg_sleep(1, 1)"))
             .hasMessageContaining("Invalid arguments in: pg_sleep(1, 1) with (integer, integer). Valid types: (double precision)");
     }
 }

